.circleci/config.yml

```diff
@@ -664,3 +675,15 @@ workflows:
     jobs:
       - build:
           context: nightly_env
+  sdk_nightly:
+    triggers:
+      - schedule:
+          # should trigger every day at 9 PM UTC (12 AM Israel Time)
```

Comment:
this is 12 AM UTC which is 3 AM Israel time (also fix the comment in the regular nightly)